VirtoCommerce.Storefront.Model/Security/Contracts/DeleteUserResponseDto.cs

```diff
@@ -0,0 +1,7 @@
+namespace VirtoCommerce.Storefront.Model.Security.Contracts
+{
+    public class DeleteUserResponseDto
+    {
+        public bool DeleteUser { get; set; }
```

Comment:
1. the returned type is SecurityResultDto, not bool
2. no need for dedicated ResponseDto class in here. Create and use more general SecurityResponseDto

Reply:
look at down